services/utils/formatDate.js

```diff
@@ -0,0 +1,8 @@
+export function formatDate(date, locale = "en-GB") {
+  const options = {
+    dateStyle: "full",
```

Comment:
Not all usage is `full`, for example here https://github.com/EddieHubCommunity/LinkFree/pull/7978/files#diff-bd5cb53f7f62529ac08cd24699132a88e8567b73d93b5289764cfa29442b5720L99 it is `short`

Reply:
okay! But I am confused on this part such that it can be both long and short?

Reply:
Also, I cannot test this how my code is working on statistic page?

Reply:
Why can't you? You can setup GitHub OAuth for your dev environment https://linkfree.io/docs/environments/environment-variables